styles_.css

```diff
@@ -0,0 +1,1069 @@
+* {
+  box-sizing: inherit;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 16px;
+  vertical-align: baseline;
+}
+
+ul {
+  list-style-type: none;
+}
+
+body {
+  background-color: #f7f7f9;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+/* Navbar and toggle bar */
+
+.togglebar {
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  z-index: 2;
+  padding: 0  20px;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
+
+.mylogo {
+  top: 39px;
+  width: 120px;
+  height: 20px;
+  position: relative;
+  font-family: "Poppins", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 20px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.mynavbar {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  left: -5%;
+  position: relative;
+}
+
+.mynavbar-ul {
+  display: none;
+  justify-content: space-between;
+}
+
+.hamburgeur {
+  height: 1.25rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0.5rem;
+}
+
+/* mypage styling */
+
+.mypage {
+  height: 126vh;
+  padding-top: 100px;
+  background-image: url("img/mobile.png");
+  background-repeat: no-repeat;
+  background-color: white;
+  border-bottom-left-radius: 80px 80px;
+}
+
+.mypage-container {
+  position: relative;
+  width: 50%;
+  border-bottom-left-radius: 2px;
+  display: flex;
+  flex-direction: column;
+}
+
+.mypage-title-head {
+  margin: 15px;
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
+}
+
+.mypage-title-dsc {
+  height: 146px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: #344563;
+}
+
+.mypage-title-connect {
+  top: 5px;
+  margin: 15px;
+  width: 121px;
+  height: 6px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.mylink {
+  margin: 15px;
+  padding-left: 0;
+  width: 170.17px;
+  height: 20px;
+  display: flex;
+  list-style-type: none;
+}
+
+.mylink-portfolio {
+  padding-right: 17px;
+  font-size: 10px;
+}
+
+.mylink-portfolio:hover {
+  transform: translateY(-10px);
+}
+
+/* mycard styling */
+
+.mycard {
+  width: 100%;
+  top: -5px;
+  position: relative;
+  background: #f7f7f9;
+}
+
+.mycard-container {
+  padding-top: 116px;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  padding-bottom: 96px;
+  row-gap: 20px;
+}
+
+.mycard-place-holder {
+  width: 295px;
+  height: auto;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+}
```

Comment:
Kindly ensure that the cards are close to the design as possible. Consider the comparison below:

Your design:
![image](https://user-images.githubusercontent.com/20554583/222940171-e109898a-7090-4403-b3e1-40a36aa74c46.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/222940194-8378ed87-6b15-43c1-bc75-894bad21c88f.png)